src/ivc/fold_relaxed_plonk_instance_chip.rs

```diff
@@ -857,114 +857,151 @@ mod tests {
             .unwrap();
     }
 
+    const LENGHT: usize = 5;
+    const TEST_FOLD_ROUND: usize = 3;
```

Comment:
add comment to explain this variable

Reply:
If it's unclear, rather pick the best names so that based on them, the purpose is clear. WIP

Reply:
This is NBD.  At least not clear to me until I saw where it is used. Something `NUM_FOLDS`?

Reply:
By just looking at the name 'TEST_FOLD_ROUNDS = n', it means a circuit is folded 'n' times to me